content/about/about_people.rst

```diff
@@ -309,3 +309,40 @@ EMH333
 .. raw:: html
 
    <br/>
+
+`Ryan Kennedy`_
+----------------
+
+.. image:: /images/r.jpg
+    :width: 110px
+    :align: right
+    :alt: Ryan Kennedy
+
+.. class:: no-breaks
+
+  **Student Systems Engineer**
+
+Ryan joined the OSL in June 2020 and is studying Computer Science with a focus on cybersecurity.
+He first learned of the OSL through Khuong and other OSL alumni during a Cyber Defense Competition.
+Ryan is the lab manager for the OSU Security Club (OSUSEC), and hopes to be able to use the knowledge he gains at the O\
+SL to improve the club's systems.
+He also loves playing video games, learning how to build exciting things, and participating in competitions with the Se\
+curity Club. 
```

Comment:
Can you please fix this wrap? You shouldn't have anything like ``\``  at the end.